src/lib/cf/cf.test.data.ts

```diff
@@ -823,8 +823,8 @@ export const userRolesForSpace = `{
   "resources": [
     {
       "metadata": {
-        "guid": "uaa-id-363",
-        "url": "/v2/users/uaa-id-363",
+        "guid": "uaa-id-253",
```

Comment:
I am really sorry you had to deal with this...

This, is possibly the worst thing about pazmin... We were hoping to introduce `jest` and `mock` which would not require the data to exist... This didn't happen due to prioritisation...

If you decide to keep this app alive and have capacity, I would suggest doing that sooner rather than later for a more pleasant experience...